book/src/ui/introduction.md

```diff
@@ -0,0 +1,40 @@
+# Introduction 
+
+The most usuall way to have your players interact with your game is through your user interface or UI.
+The UI includes all kinds of widgets from images, buttons, progress bars, text, sliders, popup menus, etc.
```

Comment:
I'm not sure we should mention progress bars, sliders, and popup menus, since amethyst doesn't support that yet.

Reply:
I was just referring to ui in general.